src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
Reviewing the code and wrapping is on `wrapValue` method on `Schema\Grammars\OracleGrammar` class. However, I think lower case is not enforced on that method. We just enclosed the given reserved word with double quote. I think I'm just used to using lower case on my migration which is why I force this to lower case.

```
    protected function wrapValue($value)
    {
        if ($this->isReserved($value)) {
            return parent::wrapValue($value);
        }

        return $value !== '*' ? sprintf($this->wrapper, $value) : $value;
    }
```